server/src/engineModel/EngineModel.ts

```diff
@@ -92,6 +93,27 @@ export class EngineModel {
                 this.completionProvider.addCompletionItem(method);
             });
         }
+
+        /* A class (or enum) with instances has the following format:
+        <class ident="EnumClassName">
+            <extends>yseop.lang.Enum</extends>
+            <instances count="2">
+                <item ident="EnumInstance_1"></item>
+                <item ident="EnumInstance_1"></item>
+            </instances>
+        </class>
```

Comment:
Lol, your comment makes it even more strange. I’d expect the `[0]` to directly provide access to the first item.

Reply:
`yclass.instances` is the array of all `instances` tags, even if there is just one element.
`yclass.instances[0].item` is the array of all `item` tags in the first `instances` tag.

Reply:
And why do we focus so much on the first instance, again?
I feel like I’ve asked that in the past but I can’t seem to recall anything so I’ll keep asking for more comments until I get it xD

Reply:
We focus on the first element in `instances` because there is only one xD

Reply:
Gosh, like, we can’t prevent the parser or whatever from creating an array but in the end it places all instances within a single `instances` element? This framework or whatever it is seems so weird.